NumberNinjaServer/routes/index.js

```diff
@@ -0,0 +1,9 @@
+var express = require('express');
+var router = express.Router();
+
+/* GET home page. */
+router.get('/', function(req, res, next) {
+  res.render('index', { title: 'Express' });
```

Comment:
You want the title to be "Express" ?

Reply:
This is just the default api to check whether express server is accessible. Hence let the title be Express.